static/js/redux/reducers/mock_modal_reducer.jsx

```diff
@@ -0,0 +1,12 @@
+import * as ActionTypes from '../constants/actionTypes';
```

Comment:
You'll need to remove all references to the onboarding tasks before this can be merged later.